test/helpers/acl.js

```diff
@@ -1,5 +1,3 @@
-// Converts an ACL expressed as a plain old object into an ACL Map
-const objToAcl = (obj) => new Map(pairs(obj).map(e => [e[0], new Map(pairs(e[1]))]))
 const pairs = (obj) => Object.keys(obj).map(k => [k, obj[k]])
-
-module.exports.objToAcl = objToAcl
+// Converts an ACL expressed as a plain old object into an ACL Map
+export const objToAcl = (obj) => new Map(pairs(obj).map(e => [e[0], new Map(pairs(e[1]))]))
```

Comment:
note: these functions are extremely convoluted. It would be nice to clear this up and make it more maintainable (not for this PR)

Is there a reason we only need these in test/helpers? Can we mock whatever logic is in src? Not creating a tracking item for this because maybe these are dumb irrelevant questions

Reply:
I believe everything related to ACK / ipfs-proxy related to deprecated window.ipfs can be removed.